### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by Aliisza** [loathers/garbage-collector#1939]
After executing the sandworm free fights, on line 2062 of fights.ts garbo will fold a wad of used tape. But then, assuming you also have apriling band helmet. Garbo will then go into doing those 3 fights, and try to refold tote back into a broken champagne bottle.

**Comment by Tokoeka** [comment]
To my knowledge you dont lose charges when folding items unless you are folding an item that has charges left from the previous day? 

Are you just reporting this with regards the inefficiency of folding and then refolding the Tote? or is there something I'm missing?

**Comment by horrible-little-slime** [comment]
I agree with asmo; it's not clear to me based on this post what suboptimal thing is happening or what optimal thing isn't happening. Could you describe what the current behavior is, what the desired behavior is, what the difference between those two is, and why that behavior is desired?
